packages/blockchain-wallet-v4/src/types/KVStoreEntry.js

```diff
@@ -33,7 +33,7 @@ export const magicHash = KVStoreEntry.define('magicHash')
 export const address = KVStoreEntry.define('address')
 export const signKey = KVStoreEntry.define('signKey')
 export const encKeyBuffer = KVStoreEntry.define('encKeyBuffer')
-export const encKeyBufferPadded = KVStoreEntry.define('encKeyBufferPadded')
+export const encKeyBufferPadded = KVStoreEntry.define('encKeyBufferUnpadded')
```

Comment:
Shouldn't this be 
`export const encKeyBufferUnpadded = KVStoreEntry.define('encKeyBufferUnpadded') `